src/codegen/trampoline.rs

```diff
@@ -24,9 +24,8 @@ pub fn generate(
     env: &Env,
     analysis: &Trampoline,
     in_trait: bool,
-    object_name: &str,
+    prepend: &str,
```

Comment:
IMHO here better be `indent: usize` as it public function and `prepend` can stay in inner privates.

Reply:
I don't insist

Reply:
Hm, all `version_condition` in this file need indent

Reply:
Ok, I'll udpate it!